gulpfile.js

```diff
@@ -15,8 +15,12 @@ const request = require('request');
 const rev = require('gulp-rev-all');
 const svg2png = require('gulp-svg2png');
 
-const criticalCssPageTypes = require('./critical-css.json');
-
+let criticalCssPageTypes = {};
+try {
+    criticalCssPageTypes = require('./critical-css.json');
+} catch (exception) {
+    // Do nothing.
```

Comment:
Even better is having a separate Gulpfile, but involves changing the formula so let's wait.